### DISCUSSION TITLE
jekyll-blog-github-pages

### DISCUSSION THREAD

**Discussion by giscus** [yshmarov/yshmarov.github.io#21] [General]
# jekyll-blog-github-pages

Are you even a developer, if you don’t use Git and Markdown for your blog?

https://blog.corsego.com/jekyll-blog-github-pages

**Comment by alec-c4** [comment]
Hey! I'd like to propose an alternative to Jekyll - https://astro.build The reason is Astro is framework-agnostic, fast as hell and there are a lot of plugins and themes which you can use to customize your blog. As an alternative to GitHub pages, you can use Vercel.com or Netlify.com - both have free plans and good enough as a hosting platform for blogs. I ran my blog alec-c4.com using Vercel and Astro and it works fine.

**Reply by yshmarov** [reply]
thanks for the suggestion! I will consider it for the next website
